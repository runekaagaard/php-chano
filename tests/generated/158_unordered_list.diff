001- <li>States
001+ Fatal error: Call to undefined method DtlIter::unordered_list() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/158_unordered_list.php on line 16
002- 	<ul>
003- 		<li>Kansas
004- 		<ul>
005- 			<li>Lawrence</li>
006- 			<li>Topeka</li>
007- 		</ul>
008- 		</li>
009- 		<li>Illinois</li>
010- 	</ul>
011- 	</li>