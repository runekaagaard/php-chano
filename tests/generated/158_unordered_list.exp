<li>States
	<ul>
		<li>Kansas
		<ul>
			<li>Lawrence</li>
			<li>Topeka</li>
		</ul>
		</li>
		<li>Illinois</li>
	</ul>
	</li>