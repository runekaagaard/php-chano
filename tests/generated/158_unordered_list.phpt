--TEST--
A generated testfile for the "unordered_list" filter.
--FILE--
<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => array (
  0 => 'States',
  1 => 
  array (
    0 => 'Kansas',
    1 => 
    array (
      0 => 'Lawrence',
      1 => 'Topeka',
    ),
    2 => 'Illinois',
  ),
))));
foreach ($items as $i) echo $i->input->unordered_list();
--EXPECT--
	<li>States
	<ul>
		<li>Kansas
		<ul>
			<li>Lawrence</li>
			<li>Topeka</li>
		</ul>
		</li>
		<li>Illinois</li>
	</ul>
	</li>