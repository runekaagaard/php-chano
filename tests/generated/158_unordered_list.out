Fatal error: Call to undefined method DtlIter::unordered_list() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/158_unordered_list.php on line 16